substrate/validator-sets/pallet/src/lib.rs

```diff
@@ -106,7 +111,7 @@ pub mod pallet {
     #[inline]
     fn in_active_serai_set(account: Public) -> bool {
       // TODO: This is bounded O(n). Can we get O(1) via a storage lookup, like we do with InSet?
-      for validator in pallet_session::Pallet::<T>::validators() {
+      for (validator, _) in Self::serai_validators() {
```

Comment:
Hm. Babe::is_member exists, yet it's also O(n) and not O(1). Possibly still best to outsource to it *with* a TODO of "TODO: is_member is internally O(n). Update Babe to use an O(1) storage lookup?"?